circom-prover/src/lib.rs

```diff
@@ -20,10 +19,10 @@ impl CircomProver {
     pub fn prove(
         proof_lib: ProofLib,
         wit_fn: WitnessFn,
-        inputs: HashMap<String, Vec<String>>,
+        input_str: String,
```

Comment:
I think it can be called `json_input_str`

Reply:
fixed in[35305a1](https://github.com/zkmopro/mopro/pull/343/commits/35305a125cf40fff17b1e3c2d640ce5032c79f8d)